articles/migration/updating_versions.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Updating Versions
+description: A guide on updating MonoGame when a new release is pushed.
+---
+
+When a new release of MonoGame is pushed, developers will need to manually update their development environment and/or existing project to make use of the new version.  This guide is intended to walk you through the steps of performing these updates.
+
+# Update dotnet Templates
+Developers will need to update the MonoGame C# Templates used to create new projects.  Doing this depends on if you are using Visual Studio 2022 or through the dotnet cli.
+
+## Visual Studio 2022
+Develoeprs using Visual Studio 2022 should be using the [MonoGame C# Project Templates](https://marketplace.visualstudio.com/items?itemName=MonoGame.MonoGame-Templates-VSExtension) extension.  This provides not only the tempaltes but also the functinality to open the *MonoGame Content Builder Editor* (MGCB Editor) within Visual Studio.  You can update the extension by performing the following
+
+1. Open Visual Studio 2022
+2. In the lanuch window, choose the *Continue without code* option at the bottom on the right.
+3. From the top menu choose, *Extensions > Manage Extensions*  to open the *Extension Manager* panel.
+4. Click the *Updates* tab and choose *MonoGame Framework C# project templates* and update it to the latest version.
+
+If prompted to close Visual Studio to finish the update, do so now to continue installing the update.
+
+## dotnet CLI
+Develoeprs using the dotnet CLI with environments such as Visual Studio Code, JetBrains Rider, or other editors, can exceute the following command in a command prompt/terminal to update the templates
+
+```sh
+dotnet new uninstall MonoGame.Templates.CSharp
+dotnet new install MonoGame.Templates.CSharp
+```
+
+The templates must be uninstalled first before the updated tempaltes are installed; otherwise, you will see warnings about template naming conflicts between the two versions.
```

Comment:
Not true, I updated by simply running `dotnet new install`

If you have experienced this issue, then also add a note/warning that if you do encounter an issue you can uninstall/reinstall.

But by default, users should ONLY need to run install.

Reply:
Moved it to note section